### PR TITLE
i18n(es): Update `api-reference.mdx` & `view-transitions.mdx`

### PR DESCRIPTION
**Pull request by Waxer59**
#### Description (required)

`api-reference.mdx` [`2be7e9f`](https://github.com/withastro/docs/commit/2be7e9f20c3b2550cef9a2747ec303a2c6c24fe2), [`3433810`](https://github.com/withastro/docs/commit/34338107bd809b9255fed094f14dbbd22e92b084) & [`a9a2779`](https://github.com/withastro/docs/commit/a9a27799a59aa390931e48ed1ca39693efc6eea4)

`view-transitions` Fix typo

### CODE REVIEW

File: src/content/docs/es/reference/api-reference.mdx [hunk 1]
```diff
@@ -1127,6 +1176,130 @@ Este componente proporciona resaltado de sintaxis específico con el lenguaje de
...
+import { Image } from 'astro:assets';
+import myImage from "../assets/my_image.png"; // La imagen es 1600x900
+---
+<!-- `alt` es obligatorio en el componente Image -->
+<Image src={myImage} alt="Una descripción de mi imagen." />
+```
+
+```html
+<!-- Salida -->
+<!-- La imagen está optimizada y se aplican atributos adecuados. -->
```
Comment (Waxer59) [inline src/content/docs/es/reference/api-reference.mdx:1193]:
```suggestion
<!-- La imagen está optimizada y se aplican atributos adecuados -->
```

File: src/content/docs/es/reference/api-reference.mdx [hunk 2]
```diff
@@ -1127,6 +1176,130 @@ Este componente proporciona resaltado de sintaxis específico con el lenguaje de
...
+    <title>Mi página de inicio</title>
+    <ViewTransitions />
+  </head>
+  <body>
+    <h1>¡Bienvenido a mi página web!</h1>
+  </body>
+</html>
+```
+
+Obtén más información sobre cómo [controlar el enrutador](/es/guides/view-transitions/#control-de-enrutador) y [añadir directivas de transición](/es/guides/view-transitions/#directivas-de-transición) a elementos de página y componentes.
```
Comment (Waxer59) [inline src/content/docs/es/reference/api-reference.mdx:1301]:
```suggestion
Mira más acerca de cómo [controlar el enrutador](/es/guides/view-transitions/#control-de-enrutador) y [añadir directivas de transición](/es/guides/view-transitions/#directivas-de-transición) a elementos de página y componentes.
```

File: src/content/docs/es/reference/api-reference.mdx [hunk 3]
```diff
@@ -1094,6 +1130,19 @@ import { Code } from 'astro:components';
 
 Este componente proporciona resaltado de sintaxis para bloques de código en el momento de la compilación (no incluye JavaScript del lado del cliente). El componente funciona internamente con Shiki y es compatible con todos los [temas populares](https://github.com/shikijs/shiki/blob/main/docs/themes.md) y [lenguajes de programación](https://github.com/shikijs/shiki/blob/main/docs/languages.md). Además, puedes agregar temas y lenguajes de programación personalizados modificando `theme` y `lang` respectivamente.
 
+### `<Fragment />`
+
+Un componente utilizado con las [directivas `set:*`](/en/reference/directives-reference/#sethtml) para representar contenido HTML sin elementos de envoltura adicionales:
```
Comment (Waxer59) [inline src/content/docs/es/reference/api-reference.mdx:1135]:
```suggestion
Un componente utilizado con las [directivas `set:*`](/es/reference/directives-reference/#sethtml) para representar contenido HTML sin elementos de envoltura adicionales:
```

File: src/content/docs/es/reference/api-reference.mdx [hunk 4]
```diff
@@ -1094,6 +1130,19 @@ import { Code } from 'astro:components';
...
+
+```astro title="src/components/SetHtml.astro" "Fragment"
+---
+const htmlString = '<p>Contenido HTML en bruto</p>';
+---
+<Fragment set:html={htmlString} />
+```
+
+Consulta más información sobre [el uso de fragmentos](/es/core-concepts/astro-syntax/#fragments) en la sintaxis de Astro.
```
Comment (Waxer59) [inline src/content/docs/es/reference/api-reference.mdx:1144]:
```suggestion
Consulta más información sobre [el uso de fragmentos](/es/core-concepts/astro-syntax/#fragmentos) en la sintaxis de Astro.
```

File: src/content/docs/es/reference/api-reference.mdx [hunk 5]
```diff
@@ -1127,6 +1176,130 @@ Este componente proporciona resaltado de sintaxis específico con el lenguaje de
...
+    <title>Mi página de inicio</title>
+    <ViewTransitions />
+  </head>
+  <body>
+    <h1>¡Bienvenido a mi página web!</h1>
+  </body>
+</html>
+```
+
+Mira más acerca de cómo [controlar el enrutador](/es/guides/view-transitions/#control-de-enrutador) y [añadir directivas de transición](/es/guides/view-transitions/#directivas-de-transición) a elementos de página y componentes.
```
Comment (Waxer59) [inline src/content/docs/es/reference/api-reference.mdx:1301]:
```suggestion
Mira más acerca de cómo [controlar el enrutador](/es/guides/view-transitions/#controlar-el-enrutador) y [añadir directivas de transición](/es/guides/view-transitions/#directivas-de-transición) a elementos de página y componentes.
```

**Review by dreyfus92 — APPROVED** [review]
LGTM ✅

**Review by Yan-Thomas — APPROVED** [review]
LGTM! Thanks peeps 🚀

**Pull request merged**